internal/server/spanner/query.go

```diff
@@ -309,16 +323,16 @@ func (sc *SpannerClient) GetNodeEdgesByID(ctx context.Context, ids []string, arc
 	var template string
 	switch arc.Out {
 	case true:
-		if arc.Decorator == "+" {
-			template = statements.getChainedEdgesBySubjectID
+		if arc.Decorator == CHAIN {
+			template = fmt.Sprintf(statements.getChainedEdgesBySubjectID, MAX_HOPS)
```

Comment:
nit: Consider formatting the string with constant args (like `MAX_HOPS`) when defining the statement itself.

Reply:
Done